packages/synapse-react-client/src/components/CitationPopover/CitationPopover.tsx

```diff
@@ -0,0 +1,267 @@
+import { useState } from 'react'
+import {
+  Button,
+  Popover,
+  Typography,
+  Box,
+  Stack,
+  FormControl,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Alert,
+} from '@mui/material'
+import CopyToClipboardIcon from '../CopyToClipboardIcon'
+import CloseIcon from '@mui/icons-material/Close'
+import { KeyboardArrowDown } from '@mui/icons-material'
+import { useCitation } from './useCitation'
+import { ReactComponent as DoubleQuotes } from '../../assets/icons/DoubleQuotes.svg'
+import { createLinkAndDownload } from './CitationPopoverUtils'
+
+type CitationPopoverProps = {
+  doi: string | undefined
+  title?: string
+  boilerplateText?: string
+}
+
+function CitationPopover({
+  doi,
+  title,
+  boilerplateText,
+}: CitationPopoverProps) {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null)
+  const [citationFormat, setCitationFormat] = useState('bibtex')
+
+  const selectSx = {
```

Comment:
add type, consider pulling this object out of the function component (or using `styled` from `'@mui/material'`)
```suggestion
  const selectSx: SxProps = {
```